app/dashboard/migrations/0117_bounty_bounty_reserved_for_user.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.1.2 on 2018-11-15 16:36
+
+from django.db import migrations, models
```

Comment:
These migrations need deleted and recreated.

Reply:
@mbeacom Should be fine now